Update docs for contacts.yaml (#281)
canonical/oci-factory: README.md

| notify | False | Dict[str, List[str]] | Who and how to notify about the build progress. |
| notify.emails | False | List[str] | List of emails that should receive notifications. |
| notify.mattermost-channels | False | List[str] | Mattermost channel **IDs**. |

> | maintainers | False | List[str] | List of maintainers' GitHub usernames, who are permitted to trigger workflows against this image. |

Reviewer: ```suggestion
| maintainers | False | List[str] | The maintainers' GitHub usernames who can trigger workflows against this image. |
```

  mattermost-channels:
    - fbdezwkcxpfofpysjore1wpfoc

maintainers:

>   - foo

Reviewer: ```suggestion
  - octocat
```

Just to be consistent with the GH user docs :p
